src/mlpack/prereqs.hpp

```diff
@@ -15,13 +15,6 @@
 #define _USE_MATH_DEFINES
 #include <cmath>
 
-// First, check if Armadillo was included before, warning if so.
-#ifdef ARMA_INCLUDES
-#pragma message "Armadillo was included before mlpack; this can sometimes cause\
- problems.  It should only be necessary to include <mlpack/core.hpp> and not \
-<armadillo>."
-#endif
```

Comment:
So, very technically, we can't do this yet because of those typedefs that are still present in `arma_extend.hpp`.  We'll be able to remove this in a future PR when that is handled.

Reply:
I think we should re-add this comment here, since we can't quite remove it yet. :+1: